pkg/option/config.go

```diff
@@ -242,6 +242,10 @@ const (
 	// ("snat", "dsr" or "hybrid")
 	NodePortMode = "node-port-mode"
 
+	// NodePortAlg indicates which algorithm is used for backend selection
+	// ("random" or "maglev")
+	NodePortAlg = "node-port-algorithm"
```

Comment:
Nit, the functionality is not specific for node port, is it? We should think on a different naming that does not hav e"node-port" on it.

Reply:
Right, I agree generally; I used the prefix here to be consistent with the existing knobs we have, e.g. see code above. I think one option would be to add proper aliases for all of them and expose those to the agent's help. One thing I want to avoid is to add yet another naming scheme just for a few which gets even more confusing.